ehsm-service/test_ehsm_service.py

```diff
@@ -0,0 +1,85 @@
+import requests 

```

Comment:
It is a little different from what we want. It tests every API and checks the return value of 200, but the functions of use are not tested. 
We'd like the case is:
1. test_aes.py:
1). CreateKey(AES)
2). Encrypt("123456")
3). Decrypt(ciphertext)
4). Check the result with the plaintext "123456"

And seems most parameters are hardcoded, so that leaves some todos in /*comments*/.
Suggest moving the file to ehsm/test/*.py 
There are two purposes of the test .py: 1. test the restful API and the functionalities 2. Show the usage of the rest APIs.

Reply:
It will be fix in the next PR.